plugin/coc-quick-lint-js/README.md

```diff
@@ -0,0 +1,15 @@
+# coc-quick-lint-js
```

Comment:
Could this be part of the Vim plugin (`plugin/vim/`)? Or does coc.nvim require its own infrastructure separate from Vim's and Neovim's for plugin management?

Ideally, this plugin would work out-of-the-box with little or no configuration if you install quick-lint-js with a system package manager, assuming you already have coc.nvim configured. This is the case with the ALE-based Vim plugin already.

Reply:
No. Like VSCode plugin is published to https://marketplace.visualstudio.com/items?itemName=quick-lint.quick-lint-js, coc-quick-lint-js can be published to npmjs.com, then user can install it by `:CocInstall coc-quick-lint-js`.

Reply:
In that case, let's keep it in its own directory (`plugin/coc-quick-lint-js/`).

I think we shouldn't include "quick-lint-js" in the directory name. Could we call it `plugin/coc.nvim/`?

Reply:
After I fix the TextEncoder error, I'll improve this.